docs/setup/new_environment.md

```diff
@@ -310,6 +312,11 @@ If you ever reboot this machine, make sure to follow the [after reboot procedure
 $ commcare-cloud monolith after-reboot all
 ```
 
+
+### In Order to set up valid SSL certificates 
+If you wish to set up valid SSL certificate please follow instructions provide in this link (../services/nginx/ssl.md)
```

Comment:
Setting up SSL isn't optional, so I'd like to integrate it fully into this new environment process.  Instead of linking to another documentation page, can you paste in only the relevant commands here?  With the change in https://github.com/dimagi/sample-environment/pull/8, step 1 should be taken care of automatically with the initial server deploy, so I think all that's necessary is setting fake_ssl_cert to no and running the commands from step 2 and 3.

Reply:
Sure, Ethan. Will update that.

Reply:
@esoergel updated the document. Please review.